.angular-cli.json

```diff
@@ -23,6 +23,7 @@
         "styles.scss"
       ],
       "scripts": [
+        "../node_modules/popper.js/dist/umd/popper.min.js"
```

Comment:
Not sure if popper is used directly in the code yet but it may need to be added to the globals object in `rollup.config.js`

Reply:
Yup, I'm getting the same error on popper.js as well.  Not able to compile and test without it.